packages/lesswrong/components/users/SubscriptionsList.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { ReactNode } from "react";
```

Comment:
This is factored out of `ViewSubscriptionsList.tsx` - the only change it to add the usage of `useCountItemsContext` to keep track of the number of subscriptions that are loaded.